features/filesystem/sd/SDBlockDevice.h

```diff
@@ -127,7 +127,7 @@ class SDBlockDevice : public BlockDevice {
      */
     virtual void debug(bool dbg);
 
-private:
+protected:
```

Comment:
Sorry about my delay.

1. Yes, I think we should expose a standard api for changing the clock speed, either the constructor (similar to serial), or the frequency method, or both.

2. @TomoYamanaka also has a valid point around the `_cs` and `_cmd` functions. It looks like the iSDIO library uses these functions to access the underlying SD protocol. These should probably be converted to public functions with the appropriate api+documentation if we want to provide these operations to users.

That being said, the two above points will take a bit of work to make sure we're exposing a reasonable api.

As a temporary solution we can just change the access to protected (accept the pr), so that the iSDIO and other libraries can adopt the library, and move them back to private access when we have replacement functions.

@0xc0170 what do you think of taking this route for now?